pkg/indexers/apiexport.go

```diff
@@ -72,3 +76,17 @@ func IndexAPIExportByClaimedIdentities(obj interface{}) ([]string, error) {
 	}
 	return sets.List[string](claimedIdentities), nil
 }
+
+// IndexAPIExportEndpointSliceByAPIExportFunc indexes the APIExportEndpointSlice by their APIExport's Reference Path and Name.
+func IndexAPIExportEndpointSliceByAPIExport(obj interface{}) ([]string, error) {
+	apiExportEndpointSlice, ok := obj.(*apisv1alpha1.APIExportEndpointSlice)
+	if !ok {
+		return []string{}, fmt.Errorf("obj %T is not an APIExportEndpointSlice", obj)
+	}
+
+	path := logicalcluster.NewPath(apiExportEndpointSlice.Spec.APIExport.Path)
+	if path.Empty() {
+		path = logicalcluster.From(apiExportEndpointSlice).Path()
```

Comment:
don't we need both ways to address it?

Reply:
Depends on how you look it up later in the code.

Reply:
Not sure I understand this comment.